locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, k8s, openshift etc
```

Comment:
shouldn't the kubernetes service name be `kubernetes` instead of `k8s`?
```
A service provided by sccli or systemd. For example:
sccli service like docker, kubernetes, openshift etc
```